libs/expo/betterangels/src/lib/hooks/newRelic/useNewRelic.ts

```diff
@@ -0,0 +1,34 @@
+import NewRelic from 'newrelic-react-native-agent';
+import { useEffect } from 'react';
+import { Platform } from 'react-native';
+
+export default function useNewRelic(version?: string, runtimeVersion?: string) {
+  useEffect(() => {
+    if (!version || !runtimeVersion) {
+      return;
+    }
+    const appToken = process.env['EXPO_PUBLIC_NEW_RELIC_MOBILE_LICENSE_KEY'];
```

Comment:
We'll need to create an env var for the app and platform.  The newrelic integrations creates keys per platform and app.

for example

```
import NewRelic from 'newrelic-react-native-agent';
import * as appVersion from './package.json';
import {Platform} from 'react-native';

// Platform-specific App Tokens
const APP_TOKENS = {
  ios: 'ios key',
  android: 'android key',
};
...
NewRelic.startAgent(APP_TOKENS[Platform.OS],agentConfiguration);
NewRelic.setJSAppVersion(appVersion.version);
AppRegistry.registerComponent(appName, () => App);
```